### ISSUE TITLE
Unable to import .rapidstart file - The property 'Code' cannot be found on this object

### ISSUE THREAD

**Issue by Drakonian** [microsoft/navcontainerhelper#3051]
**Describe the issue**
BCCContainerHelper 5.0.2
The Get-PackageInfoFromRapidStartFile function contains a bug that makes it impossible to import a Package via the Run-AlPipeline function. 
I am attaching a pull request with the fix. Even if you see more correct fix at least you know problem place :)

**Scripts used to create container and cause the issue**
```
UploadImportAndApply-ConfigPackageInBcContainer
```

**Full output of scripts**
```
UploadImportAndApply-ConfigPackageInBcContainer Telemetry Correlation Id: 46be8886-fedb-4de8-a978-7ee1ee78bf6d
##[error]The property 'Code' cannot be found on this object. Verify that the property exists.
At C:\Program Files\WindowsPowerShell\Modules\BcContainerHelperUK\5.0.2\ConfigPackageHandling\UploadImportAndApply-Conf
igPackageInBcContainer.ps1:46 char:21
+                     $packageId = $packageInfo.Code
+                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], PropertyNotFoundException
    + FullyQualifiedErrorId : PropertyNotFoundStrict
##[error]PowerShell exited with code '1'.

...
```

**Comment by freddydk** [comment]
Thanks for the help, much appreciated.

**Comment by freddydk** [comment]
Shipped in 5.0.3
